coalib/bearlib/aspects/Formatting.py

```diff
@@ -391,5 +392,7 @@ class docs:
         Choosing a preferred quotation and using it everywhere (if possible).
         """
     preferred_quotation = Taste[str](
-        'Represents the preferred quotation',
-        ('\'', '"'), default='\'')
+        'Represents the preferred quotation marks.'
+        'It ensures that every string contains the selected style of quotes,'
+        'except where there is a delimiter collision.',
+        ('\'', '"'), default='"')
```

Comment:
Please create an issue, and explain why the default is changing from `'` to `"`

"as discussed in coala/aspects gitter thread" is not good enough.

c.f. https://github.com/coala/coala/pull/5475#issuecomment-396582740